docs: update dev guide accounting for monrepo structure

juliusgb opened the pull request.

judicaelandria commented on guides/development.md:
```diff
@@ -1,10 +1,12 @@
-- Run `yarn start` to start the development server for the package and for the example.
+- Run `yarn build:packages`
+
+- Run `yarn start:www` to start the development server for the package and for the example.
```
actually this only start the website in apps/www
yarn start will start the packages and you can navigate to localhost:5420 to see it

judicaelandria commented on guides/development.md:
```diff
@@ -1,10 +1,12 @@
+
+- Run `yarn start:www` to start the development server for the package and for the example.
 
 - Open `localhost:5420` to view the example project.
```
so this would be localhost:3000

juliusgb commented:
Thanks for the review.

If I understand correctly, my changes make the guide incorrect :sweat_smile:

I've changed it back and added the build step.

judicaelandria commented:
> Thanks for the review.
> If I understand correctly, my changes make the guide incorrect :person_facepalming:
not really! you just need to add yarn start:www for starting the web app at apps/www
and yarn start to start the whole  app (develop, embedded, etc)
yarn build:packages is fine

judicaelandria commented:
> > Thanks for the review.
> > If I understand correctly, my changes make the guide incorrect :person_facepalming:

not really! you just need to add yarn start:www for starting the web app at apps/www
and yarn start to start the whole  app (develop, embedded, etc) yarn build:packages is fine

steveruizok commented:
I've added a postinstall script that should make the manual `yarn build` step unnecessary—and save us a ton of time answering that question on the discord ;)

steveruizok commented:
If that was the only change, then I'm going to close this one!
